lib/build/bundle.js

```diff
@@ -0,0 +1,63 @@
+var assign = require("lodash").assign;
+var minify = require("../stream/minify");
+var devBundle = require("../stream/dev_bundle");
```

Comment:
I don't think you should throw, it would just create a bundle of your entire graph, I think.